Packs/VMRay/README.md

```diff
@@ -1 +1 @@
-*Note: Support for this pack will be moving to the partner around Oct 15, 2021.*
+*Note: Support for this pack moved to the Partner on December, 21, 2021. Please contact the Partner directly via the support link on the right.*
```

Comment:
This readme should have detailed content in it on pack value and your company (similar to the paid packs), please sync w Zac on your side for details.